src/gui/screen_lan_settings.c

```diff
@@ -82,7 +83,7 @@ static void _addrs_to_str(char *param_str, uint8_t flg) {
 
 static void _parse_MAC_addr(char *mac_addr_str) {
     uint8_t mac_addr[6] = { 0, 0, 0, 0, 0, 0 };
-    for (uint8_t i = 0; i < MAC_ADDR_SIZE; i++)
+    for (uint8_t i = 0; i < 6; i++)
```

Comment:
i would not use "6" but sizeof(mac_addr)

Reply:
It is not necessary to put in the size of the array if you initialize it. So this also works: uint8_t mac_addr[] = { 0, 0, 0, 0, 0, 0 };